modules/check_stop_condition.py

```diff
@@ -0,0 +1,57 @@
+"""
+Checks whether the drone has reached it's max flight time and sends it back to launch.
+"""
+
+import dronekit
+
+from . import upload_commands
+
+
+MAVLINK_LANDING_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL
+MAVLINK_LANDING_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_RETURN_TO_LAUNCH
+
+
+def check_stop_condition(start_time: float, current_time: float, drone: dronekit.Vehicle, MAXIMUM_FLIGHT_TIME: int)\
```

Comment:
Lowercase `MAXIMUM_FLIGHT_TIME` (it is not a global constant) and make it a `float` .